README.md

```diff
@@ -83,7 +85,7 @@ curl -X DELETE http://localhost:8083/connectors/datagen-pageviews
 The schema registry is used to encode/decode events stored in kafka.  This repository stores the schemas from the 'ec' namespace.  To modify the schemas, change the schema dsl in 'avro/dsl/*', regenerate the avsc files thru the following rake task
 ```
 bundle exec rake avro:generate
-```    
```

Comment:
@steveburkett please update your editor to remove blank spaces at the end of lines